src/ray/core_worker/core_worker.cc

```diff
@@ -1462,13 +1462,13 @@ std::string CoreWorker::OverrideTaskOrActorRuntimeEnv(
     std::vector<std::string> *runtime_env_uris) {
   std::shared_ptr<rpc::RuntimeEnv> parent = nullptr;
   if (options_.worker_type == WorkerType::DRIVER) {
-    if (serialized_runtime_env == "") {
+    if (serialized_runtime_env == "" || serialized_runtime_env == "{}") {
```

Comment:
Thinking now, can we just create a util function? Like `IsRuntimeEnvEmpty(serialized_runtime_env)` and use it all across the codebase

Reply:
This doesn't need to block this PR

Reply:
Agree, we should clean this up.